zpi-frontend/src/App.tsx

```diff
@@ -67,50 +66,49 @@ export default function App(props: IAppProps) {
                 <Route element={<RequireAuth allowedRoles={['supervisor']} />}>
                   <Route path='supervisor-reservation' element={<SupervisorReservationPage />} />
                   <Route path='my' element={<ReservationPage />} />
-                  <Route path='addthesis' element={<AddThesisPage role={'admin'} mail={'john.doe@pwr.edu.pl'} />} />
-                  <Route path='updatethesis/:thesisId' element={<UpdateThesisPage role={'employee'}
-                                                                                  mail={'john.doe@pwr.edu.pl'} />} />
+                  <Route path='sup-theses/add' element={<AddThesisPageSupervisor />} />
+                  <Route path='sup-theses/edit/:id' element={<AddThesisPageSupervisor />} />
```

Comment:
Z którego miejsca dostajemy sie teraz do doawania i edycji tematów jako prowadzący, skoro w ścieżce "my" nadal jest strona rezerwacji? A tam z tego co rozumiem powinien byc własnie ten widok, z którego prowadzący dodaje swoje tematy.

Reply:
Bardzo dobre pytanie, strona 'my' z tego co rozumiem nie jest jeszcze skończona, bo wygląda niespójnie z resztą i nie byłem pewny jak to ma wyglądać do końca, więc nie dodawałem nigdzie przycisku na to, jako że nie wiem jak to będzie się prezentować do końca. Mogę dodać, mmi to obojętne, moja myśl była taka, że się to doda jak się rozwinie tamta strona dalej

Reply:
No ona nie jest nawet zaczęta, tam na samym początku tylko było cos wstawione, żeby cos się wyświetlało, ale teraz tam powinno byc własnie to wszystko do dodawania dla pracownika i myslałam, że ty to tu masz, bo nie ma w backlogu innego zadania na to i niedobrze, że jest taka dziura, bo nie możemy dostawac sie do tego dodawania ze ścieżki to musi byc gdzies na stronie.
No więc nie wiem, jesli to nie podchodzi pod twoje zadanie to trzeba dodać do backloga nowe zadanie na to, bo to nie jest nigdzie uwzglednione w takim razie.